include/boost/fusion/container/set/set.hpp

```diff
@@ -97,39 +97,20 @@ namespace boost { namespace fusion
 
         template <typename Sequence>
         BOOST_FUSION_GPU_ENABLED
-        set(Sequence const& rhs)
-            : data(rhs) {}
-
-#if !defined(BOOST_NO_CXX11_RVALUE_REFERENCES)
-        template <typename Sequence>
-        BOOST_FUSION_GPU_ENABLED
         set(Sequence&& rhs)
             : data(std::forward<Sequence>(rhs)) {}
-#endif
 
         BOOST_CONSTEXPR BOOST_FUSION_GPU_ENABLED
         explicit
         set(typename detail::call_param<T>::type ...args)
             : data(args...) {}
 
-#if !defined(BOOST_NO_CXX11_RVALUE_REFERENCES)
         template <typename ...U>
         BOOST_CXX14_CONSTEXPR BOOST_FUSION_GPU_ENABLED
         explicit
         set(U&& ...args)
```

Comment:
Because SFINAE yields hard error in some combination with constexpr (see https://llvm.org/bugs/show_bug.cgi?id=23135), I don't want to use SFINAE as much as possible until serious regression is reported. If we decide dropping implicit sequence conversion ctor, we will be able to write maximize simpler, clearer and effective variadic containers. (IMO, most of the `implicit` often confuse users and to be complex gratuitously...)

> One last thing, is the call_param<T>::type... constructor still needed?

I ran regression and confirmed that can be dropped.